From: Ben Hutchings <ben@decadent.org.uk>
Date: Mon, 01 Feb 2016 09:05:24 +0100
Subject: usb: Fix ABI change in 4.3.5
Forwarded: not-needed

struct usb_device gained two new bitfields, but there were plenty of
padding bits to spare.  Hide them from genksyms.

---
--- a/include/linux/usb.h
+++ b/include/linux/usb.h
@@ -582,8 +582,11 @@ struct usb_device {
 	unsigned usb2_hw_lpm_enabled:1;
 	unsigned usb2_hw_lpm_allowed:1;
 	unsigned usb3_lpm_enabled:1;
+#ifndef __GENKSYMS__
 	unsigned usb3_lpm_u1_enabled:1;
 	unsigned usb3_lpm_u2_enabled:1;
+	/* 18 bits spare */
+#endif
 	int string_langid;
 
 	/* static strings from the device */
